src/sentry/models/project.py

```diff
@@ -245,6 +245,7 @@ class Project(Model, PendingDeletionMixin, OptionMixin, SnowflakeIdMixin):
     # projects that were created before this field was present
     # will have their first_event field set to date_added
     first_event = models.DateTimeField(null=True)
+    project_template = FlexibleForeignKey("sentry.ProjectTemplate", null=True)
```

Comment:
Might make sense as just `template`, otherwise we'll be calling `project.project_template` a lot